gating/pre_merge_test/run_system_tests.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+
+## Deploy virualenv for testing enironment molecule/ansible-playbook/infratest
+
+## Shell Opts ----------------------------------------------------------------
+
+set -ev
+set -o pipefail
+export ANSIBLE_HOST_KEY_CHECKING=False
+
+## Variables -----------------------------------------------------------------
+
+SYS_WORKING_DIR=$(mktemp  -d -t system_test_workingdir.XXXXXXXX)
+SYS_VENV_NAME="${SYS_VENV_NAME:-venv-molecule}"
+SYS_TEST_SOURCE_BASE="${SYS_TEST_SOURCE_BASE:-https://github.com/rcbops}"
+SYS_TEST_SOURCE="${SYS_TEST_SOURCE:-rpc-openstack-system-tests}"
+SYS_TEST_SOURCE_REPO="${SYS_TEST_SOURCE_BASE}/${SYS_TEST_SOURCE}"
+SYS_TEST_BRANCH="${SYS_TEST_BRANCH:-master}"
+SYS_CONSTRAINTS="${SYS_WORKING_DIR}/${SYS_TEST_SOURCE}/constraints.txt"
+SYS_REQUIREMENTS="${SYS_WORKING_DIR}/${SYS_TEST_SOURCE}/requirements.txt"
+SYS_INVENTORY="/opt/openstack-ansible/playbooks/inventory"
```

Comment:
Is this the proper value for the dynamic inventory for an MNAIO deploy, or is there another value or variable that should be referenced?